libcuda/cuda_runtime_api.cc

```diff
@@ -1199,22 +1199,23 @@ __host__ cudaError_t CUDARTAPI cudaGetDevice(int *device)
 	*device = g_active_device;
 	return g_last_cudaError = cudaSuccess;
 }
+
 __host__ cudaError_t CUDARTAPI cudaDeviceGetLimit ( size_t* pValue, cudaLimit limit )
 {
 	if(g_debug_execution >= 3){
             announce_call(__my_func__);
    	 }
         switch(limit) {
-        case 0:
+        case 0:  // cudaLimitStackSize
                                  *pValue=1024;
                                  break;
-        case 2:
+        case 2:  // cudaLimitMallocHeapSize
                                  *pValue=8388608;
                                  break;
-        case 3:
+        case 3: // cudaLimitDevRuntimeSyncDepth
                                  *pValue=2;
                                  break;
-        case 4:
+        case 4: // cudaLimitDevRuntimePendingLaunchCount
                                  *pValue=2048;
                                  break;
         default:
```

Comment:
These values are same for both 1080 and Volta - hence haven't added a config for each of them yet.

Reply:
We should just add the config file option now, so we don't run into problems with this again later.

Reply:
Ok - I will add them. But since I don't know these values for other architectures, should I use these values as default?

Reply:
We can run device query on the appropriate GPUs and find the values, or check on the NVIDIA specs, no?

Reply:
I was only able to run deviceQuery with GTX 1080 and Volta (the ones we have in our lab) - which I did and found them to be the same across both. I doubt we will find them in specs (did a quick google).